packages/mobile/src/store/sagas.ts

```diff
@@ -12,6 +13,7 @@ export default function* rootSaga() {
   const sagas = [
     // config
     ...backendSagas(),
+    ...analyticsSagas(),
```

Comment:
cool! so we want to have these around for now then?

Reply:
Yeah, they aren't touching any state. Don't do anything other than call amplitude essentially so it's fine to have them